### ISSUE TITLE
Mapping from website to model 

### ISSUE THREAD

**Issue by mrvirus9898** [ActoKids/AD440_W19_CloudPracticum#46]
Now that the basic mapping is done, code will need to be produced that operates on incoming data, and formats it to an ideal JSON object. Write a script that accepts a reasonable string, and converts it to a JSON that is compliant with the API teams keys. Writing to a DB will not be necessary, yet. 

Please indicate the time spent on this, any issues that you are having, any good references you found for this subject, and credit anyone helped you out.

**Comment by mrvirus9898** [comment]
Estimated time: 8 hours.
Current time spent: 2 hours.

I spent time looking into how Python handles Regex and if there are any packages that allow me to easily implement a mapping scheme.

**Comment by mrvirus9898** [comment]
Estimated time: 8 hours.
Current time spent: 12 hours.

Who needs Regex and such when python dictionaries exist! However, I did run into an issue getting beautifulsoup to install correctly into python, which is where I burned most of my time. I figured out a work around, and used Mikes crawler to pull a JSON of data from eventbrite. I saved this JSON so that I could work with it without spamming eventbrite. 

This proposed mapping scheme contains a dictionary of key-value pairs, where the key is the old key, and the value is the new key that our API team will be expecting. Mapping protocol checks the old key and returns the new, proper key if found.

To test this code, download both the JSON file and the Py file. Make sure to run the Py file in the same folder as the JSON file. 

Link to pull request: https://github.com/ActoKids/web-crawler/pull/4
